Agents/MetOfficeAgent/build_py4jps_resources.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+
+# Create directories for jps_base_lib and stack_client resource files
+# (to be imported by py4jps)
+mkdir tmp_base
+mkdir tmp_stack
+
+# Clone TWA repository
+git clone https://github.com/cambridge-cares/TheWorldAvatar.git TWA
+cd TWA
+
+# Build latest JPS_BASE_LIB .jar
+git checkout main
+git pull
+cd JPS_BASE_LIB
+mvn clean package -DskipTests
+# Copy required resource files
+cp ./target/jps-base-lib*.jar ../../tmp_base/
+cp -r ./target/lib ../../tmp_base/
+cd ..
+
+# Build latest STACK_CLIENTS .jar
+git checkout dev-MetOfficeAgent-withinStack
+git pull
+cd Deploy/stacks/dynamic/stack-clients
+mvn clean package -DskipTests
+cp ./target/stack-clients*.jar ../../../../../tmp_stack/
+cp -r ./target/lib ../../../../../tmp_stack/
+cd ../../../../..
+
+# Remove TWA repository
+rm -fr TWA
```

Comment:
This it seems very inefficient. I think a better solution will be to build and deploy the Python wrappers as part of the Java/Maven build process. That way both libraries should stay in sync. I've had a quick look into this and there is a Maven plugin that should be able to do what we need.

Reply:
> This it seems very inefficient. I think a better solution will be to build and deploy the Python wrappers as part of the Java/Maven build process. That way both libraries should stay in sync. I've had a quick look into this and there is a Maven plugin that should be able to do what we need.

I believe this is out of scope for this piece of work. Jiaru takes care of releasing new py4jps versions "every now and then", which can be directly installed from pypi. So, there is an established release process.
This script is/was only needed to install the latest (pre-released) functions from the base lib. So, normally this script should not be needed

Reply:
What happened to this script?

Reply:
> What happened to this script?

This is already on `main` as part of the approved and merged PR #381